### PR TITLE
Adding OCD IDs for Hongkong Legislative Council.

### PR DESCRIPTION
**Pull request by novitam**
Legislative Council of Hongkong is a body comprising of 70 members:
- 35 are directly elected through 5 geographical constituencies.
- 35 elected through interest-group-based functional constituencies with limited electorates.

Source: https://en.wikipedia.org/wiki/Legislative_Council_of_Hong_Kong

The functional constituencies of Hong Kong have had many changes. In this pull request, we include currently valid constituencies.

### CODE REVIEW

**Comment by novitam** [comment]
@jloutsenhizer PTAL, thanks!

File: identifiers/country-hk/functional_constituencies.csv [hunk 1]
```diff
@@ -0,0 +1,30 @@
+id,name
+ocd-division/country:hk/functional:heung_yee_kuk,Heung Yee Kuk
```
[the remark this answers is not in the document]
Comment (novitam) [reply identifiers/country-hk/functional_constituencies.csv]: Done

File: identifiers/country-hk/geographical_constituencies.csv [hunk 2]
```diff
@@ -0,0 +1,6 @@
+id,name
+ocd-division/country:hk/geographical:hk_island,Hongkong island
```
[the remark this answers is not in the document]
Comment (novitam) [reply identifiers/country-hk/geographical_constituencies.csv]: Done

**Review by jloutsenhizer — APPROVED** [review]

**Comment by novitam** [comment]
Thank you. @jpmckinney for second review.

File: identifiers/country-hk/geographical_constituencies.csv [hunk 3]
```diff
@@ -0,0 +1,6 @@
+id,name
+ocd-division/country:hk/geographical_constituency:hk_island,Hongkong island
```
Comment (jpmckinney) [inline identifiers/country-hk/geographical_constituencies.csv:2]:
```suggestion
ocd-division/country:hk/geographical_constituency:hk_island,Hong Kong Island

```

File: identifiers/country-hk/functional_constituencies.csv [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ocd-division/country:hk/functional_constituency:social_welfare,Social Welfare
+ocd-division/country:hk/functional_constituency:real_estate_and_construction,Real Estate and Construction
+ocd-division/country:hk/functional_constituency:tourism,Tourism
+ocd-division/country:hk/functional_constituency:commercial_first,Commercial (First)
+ocd-division/country:hk/functional_constituency:commercial_second,Commercial (Second)
+ocd-division/country:hk/functional_constituency:industrial_first,Industrial (First)
+ocd-division/country:hk/functional_constituency:industrial_second,Industrial (Second)
+ocd-division/country:hk/functional_constituency:finance,Finance
+ocd-division/country:hk/functional_constituency:financial_services,Financial Services
+ocd-division/country:hk/functional_constituency:sports_performing_arts_culture_and_publication,Sports; Performing Arts; Culture and Publication
```
Comment (jpmckinney) [inline identifiers/country-hk/functional_constituencies.csv:23]:
```suggestion
ocd-division/country:hk/functional_constituency:sports_performing_arts_culture_and_publication,"Sports, Performing Arts, Culture and Publication"
```

File: identifiers/country-hk/functional_constituencies.csv [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+ocd-division/country:hk/functional_constituency:agriculture_and_fisheries,Agriculture and Fisheries
+ocd-division/country:hk/functional_constituency:insurance,Insurance
+ocd-division/country:hk/functional_constituency:transport,Transport
+ocd-division/country:hk/functional_constituency:education,Education
+ocd-division/country:hk/functional_constituency:legal,Legal
+ocd-division/country:hk/functional_constituency:accountancy,Accountancy
+ocd-division/country:hk/functional_constituency:medical,Medical
+ocd-division/country:hk/functional_constituency:health_services,Health Services
+ocd-division/country:hk/functional_constituency:engineering,Engineering
+ocd-division/country:hk/functional_constituency:architectural_surveying_and_planning,Architectural; Surveying and Planning (later on Architectural; Surveying; Planning and Landscape)
```
Comment (jpmckinney) [inline identifiers/country-hk/functional_constituencies.csv:12]:
```suggestion
ocd-division/country:hk/functional_constituency:architectural_surveying_and_planning,"Architectural, Surveying, Planning and Landscape"

```

**Review by jpmckinney — COMMENTED** [review]
A few name changes.

File: identifiers/country-hk/geographical_constituencies.csv (hunk 3 above)
Comment (novitam) [reply identifiers/country-hk/geographical_constituencies.csv]: Done

File: identifiers/country-hk/functional_constituencies.csv (hunk 5 above)
Comment (novitam) [reply identifiers/country-hk/functional_constituencies.csv]: Done

File: identifiers/country-hk/geographical_constituencies.csv [hunk 6]
```diff
@@ -1,5 +1,5 @@
 id,name
-ocd-division/country:hk/geographical_constituency:hk_island,Hongkong island
+ocd-division/country:hk/geographical_constituency:hk_island,Hong Kong island
```
Comment (jpmckinney) [inline identifiers/country-hk/geographical_constituencies.csv:2]: Hmm, Island is still not capitalized, but should be. For my previous suggestions, you could have clicked "Commit suggestion".

File: identifiers/country-hk/functional_constituencies.csv [hunk 7]
```diff
@@ -9,7 +9,7 @@ ocd-division/country:hk/functional_constituency:accountancy,Accountancy
 ocd-division/country:hk/functional_constituency:medical,Medical
 ocd-division/country:hk/functional_constituency:health_services,Health Services
 ocd-division/country:hk/functional_constituency:engineering,Engineering
-ocd-division/country:hk/functional_constituency:architectural_surveying_and_planning,Architectural; Surveying and Planning (later on Architectural; Surveying; Planning and Landscape)
+ocd-division/country:hk/functional_constituency:architectural_surveying_and_planning,"Architectural, Surveying and Planning (later on Architectural, Surveying, Planning and Landscape)"
```
Comment (jpmckinney) [inline identifiers/country-hk/functional_constituencies.csv:12]: Why the "Name (later Name)" format? Why not just the current name (as in my previous suggestion)?

File: identifiers/country-hk/geographical_constituencies.csv (hunk 6 above)
Comment (novitam) [reply identifiers/country-hk/geographical_constituencies.csv]: I should have just clicked "Commit suggestion", didn't realize I could do that until after I made the edits.

File: identifiers/country-hk/functional_constituencies.csv (hunk 7 above)
Comment (novitam) [reply identifiers/country-hk/functional_constituencies.csv]: Same as above.

File: identifiers/country-hk/functional_constituencies.csv [hunk 8]
```diff
@@ -9,7 +9,7 @@ ocd-division/country:hk/functional_constituency:accountancy,Accountancy
 ocd-division/country:hk/functional_constituency:medical,Medical
 ocd-division/country:hk/functional_constituency:health_services,Health Services
 ocd-division/country:hk/functional_constituency:engineering,Engineering
-ocd-division/country:hk/functional_constituency:architectural_surveying_and_planning,"Architectural, Surveying and Planning (later on Architectural, Surveying, Planning and Landscape)"
+ocd-division/country:hk/functional_constituency:architectural_surveying_and_planning,"""Architectural, Surveying, Planning and Landscape"""
```
Comment (jpmckinney) [inline identifiers/country-hk/functional_constituencies.csv]:
```suggestion
ocd-division/country:hk/functional_constituency:architectural_surveying_and_planning,"Architectural, Surveying, Planning and Landscape"
```

File: identifiers/country-hk/functional_constituencies.csv [hunk 9]
```diff
@@ -20,7 +20,7 @@ ocd-division/country:hk/functional_constituency:industrial_first,Industrial (Fir
 ocd-division/country:hk/functional_constituency:industrial_second,Industrial (Second)
 ocd-division/country:hk/functional_constituency:finance,Finance
 ocd-division/country:hk/functional_constituency:financial_services,Financial Services
-ocd-division/country:hk/functional_constituency:sports_performing_arts_culture_and_publication,"Sports, Performing Arts, Culture and Publication"
+ocd-division/country:hk/functional_constituency:sports_performing_arts_culture_and_publication,"""Sports, Performing Arts, Culture and Publication"""
```
Comment (jpmckinney) [inline identifiers/country-hk/functional_constituencies.csv]:
```suggestion
ocd-division/country:hk/functional_constituency:sports_performing_arts_culture_and_publication,"Sports, Performing Arts, Culture and Publication"

```

**Review by jpmckinney — COMMENTED** [review]
I suggested edits to avoid having escaped quote characters in the name.

**Comment by jpmckinney** [comment]
Oops, I didn't see the recent commit. Looks good!

**Pull request merged**